llvm/test/CodeGen/AMDGPU/llvm.amdgcn.raw.ptr.buffer.store.ll

```diff
@@ -424,6 +424,150 @@ define void @buffer_store_v2i64__voffset_add(ptr addrspace(8) inreg %rsrc, <2 x
   ret void
 }
 
+define void @buffer_store_p0__voffset_add(ptr addrspace(8) inreg %rsrc, ptr %data, i32 %voffset) #0 {
+  %voffset.add = add i32 %voffset, 60
```

Comment:
Missing checks?

Reply:
For some reason this test isn't already autogenerated and I reverted too much of the generated checks